docs/changes/samp/17673.bugfix.rst

```diff
@@ -0,0 +1 @@
+samp_hub --log-level choices inferred from logging, fixes [#17670].
```

Comment:
These fragments are meant to be descriptive and as self-contained as possible. Here's one way to edit it to this effect
```suggestion
Fix setting logging level from the `samp_hub` command line.
Previously, `samp_hub --log-level OFF` was documented as supported but
actually caused an exception to be raised. `--log-level NOTSET` replaces it,
as infered from the standard library's `logging` module.
```